src/components/advanced-toolbar-control/index.js

```diff
@@ -108,18 +110,19 @@ const AdvancedToolbarControl = props => {
 				onChangeUnit={ props.onChangeUnit }
 				screens={ props.screens }
 			/>
-			<Toolbar
+			<ButtonGroup
 				{ ...omit( props, [ 'className', 'help', 'label', 'units', 'unit', 'onChangeUnit', 'screens' ] ) }
-				controls={ controls.map( option => {
-					return {
-						...option,
-						onClick: () => props.onChange( option.value !== props.value ? option.value : '' ),
-						isActive: props.value === option.value,
-						extraProps: {
-							 ...( ! option.icon ? { children: option.custom || <span className="ugb-advanced-toolbar-control__text-button">{ option.title }</span> } : {} ),
-						},
-					}
-				} ) }
+				children={
+					controls.map( ( option, index ) => {
+						const controlProps = {
+							...option,
+							onClick: () => props.onChange( option.value !== props.value ? option.value : '' ),
+							isPrimary: props.value === option.value,
+							children: ! option.icon ? option.custom || <span className="ugb-advanced-toolbar-control__text-button">{ option.title }</span> : '',
```

Comment:
Better to do `null` here since an empty string as a child might still render something

```suggestion
							children: ! option.icon ? option.custom || <span className="ugb-advanced-toolbar-control__text-button">{ option.title }</span> : null,
```

Reply:
This is noted. Thank you!